doc/superbuild/source/index.rst

```diff
@@ -201,6 +201,16 @@ SUNDIALS License and Notices
    Changelog_link.rst
    FAQ_link.rst
 
+..
+   Uncomment once all example docs are updated
```

Comment:
Should we just add this section now and add placeholders with links to the PDF versions for the other packages?